### ISSUE TITLE
Generic support 

### ISSUE THREAD

**Issue by mokone91** [nlepage/typedoc-plugin-resolve-crossmodule-references#6]
Hi!
i have a code like:
packages/base/index.ts
```
export interface List<T> {
  total: number;
  items: T[];
}
```

packages/other/index.ts
```
async listEntity(): Promise<List<Entity>>
```

But in docs i getting:
```
listEntity(): Promise<List>
```

Thanks! this plug-in is awesome!

**Comment by nlepage** [comment]
Hi @mokone91 

Is `Entity` a type parameter?

Maybe declared in a class like this:
```ts
class Repository<Entity> {
  async listEntity(): Promise<List<Entity>>
}
```

**Comment by nlepage** [comment]
@mokone91 I just published release 0.2.2 to fix this, let me know if it is OK for you.

**Comment by mokone91** [comment]
It works now, thanks!
